addons/Table/src/presenter.js

```diff
@@ -1721,6 +1721,9 @@ function AddonTable_create() {
 
     presenter.DraggableDroppableGap.prototype.fillGap = function (selectedItem) {
         DraggableDroppableObject.prototype.fillGap.call(this, selectedItem);
+        if (this.$view) {
+            this.$view.html(presenter.textParser.parseAltTexts(this.$view.html()));
```

Comment:
Gdy pierwszy raz zauważyłem ten fragment kodu to pomyślałem, że naraża ono na sytuację, że skrypty będą mogły być przez to wywoływane. Stworzyłem lekcję testową na to: https://test-pla-61-dot-mauthor-dev.ew.r.appspot.com/present/6458836225359873 , gdzie jako element 1 jest umieszczony skrypt. Po kliknięciu w ten element i po umieszczeniu go w Tabeli wywołuje się skrypt. 

Sprawdziłem też to na produkcji i jest takie same zachowanie.
Rozmawiałem o tym z Wojtkiem i najlepiej byłoby abyś spojrzał, czy nie można sanitize tu wykonać. Do tego czasu najlepiej byłoby aby Twe zadanie nie przeszło dalej. 
Na SourceListę byłoby potrzebne nowe zadanie. Jeżeli uważasz, że na to (na tabelę) też powinno być osobne zadanie to proszę daj znać. Z tego powodu tu zostawiam jedynie komentarz a Twe zadanie przestawiam na onHold.